### PR TITLE
LNX-183 Add Take action and Agent inventory

### PR DESCRIPTION
**Pull request by EasyJB**

### CODE REVIEW

**Comment by filip-szweda** [comment]
If you've decided to rename the `Action` from `Collect` to `Take` please update the story.
IMHO `Collect` is a better name, but that's entirely up to you

File: scene/world_updater/entity_deserializer/EntityDeserializer.gd [hunk 1]
```diff
@@ -10,7 +10,6 @@ func deserialize_action(attributes: Dictionary, type: String):
 	if not entity:
 		push_error("[ERROR] Deserializing unknown action type: " + type)
 		return null
-
```
Comment (Krystian030) [inline scene/world_updater/entity_deserializer/EntityDeserializer.gd:13]: enter

**Review by Krystian030 — APPROVED** [review]

File: .gut_editor_config.json [hunk 2]
```diff
@@ -39,4 +39,4 @@
  "suffix": ".gd",
  "tests": [],
  "unit_test_name": null
-}
```
Comment (filip-szweda) [inline .gut_editor_config.json:42]: Revert

File: addons/gut/utils.gd [hunk 3]
```diff
@@ -482,4 +482,4 @@ func get_scene_script_object(scene):
 
 		node_idx += 1
 
-	return to_return
```
Comment (filip-szweda) [inline addons/gut/utils.gd:485]: Revert

**Review by filip-szweda — APPROVED** [review]

**Pull request closed without merging**